integrations/helm/chartsync/chartsync.go

```diff
@@ -227,13 +225,13 @@ func getChartDirs(logger log.Logger, checkout *helmgit.Checkout) ([]string, erro
 	return chartDirs, nil
 }
 
-// newCommits ... determines if charts need to be released
+// newCommits determines if charts need to be released
 //		go-git.v4 does not provide a possibility to find commit for a particular path.
 // 		So we find if there are any commits at all sice last time
 func (chs *ChartChangeSync) newCommits() (bool, string, error) {
 	chs.logger.Log("info", "Getting new commits")
 
-	checkout := chs.release.Repo.ChartsSync
+	checkout := chs.release.Repo.ConfigSync
 
 	chs.logger.Log("info", fmt.Sprintf("Repo dir = %s", checkout.Dir))
 
```

Comment:
On line 240 below, we are using `helmgit.ChartsChangesClone` however I don't believe we are setting up this repo anymore. Should this be `helmgit.FhrsChangesClone` instead?

Reply:
You are right. Corrected.